sharding-core/sharding-core-parse/sharding-core-parse-common/src/main/java/org/apache/shardingsphere/core/parse/sql/segment/dml/column/ColumnSegment.java

```diff
@@ -65,6 +65,15 @@ public ColumnSegment(final int startIndex, final int stopIndex, final String nam
     public final String getQualifiedName() {
         return null == owner ? name : owner.getTableName() + "." + name;
     }
+
+    /**
+     * Get owner name.
+     *
+     * @return owner name
+     */
+    public final String getOwnerName() {
+        return null == owner ? null : owner.getTableName();
```

Comment:
If owner is a alias, is it possible this function returns its alias？

Reply:
This func returns the alias truly, you can check the unit test `EncryptSQLRewriteEngineParameterizedTest`

Reply:
Thanks for your reply, there are tableName and alias in `TableSegment`, when getTableName(), it should not return alias actually. But you are right, getTableName() return alias in `EncryptSQLRewriteEngineParameterizedTest` , which is an another issue.